src/app/infrastructure/computerScore.js

```diff
@@ -0,0 +1,27 @@
+export const computerScore = (level, numberOfQuestions) => {
+    if ((level === 1 || level === 2 || level === 3) && numberOfQuestions > 0) {
```

Comment:
`if(level >= 1 && level <= 3 && numberOfQuestions > 0)`

Tak chyba lepiej będzię wyglądać

Reply:
jeśli wywołasz funkcje z taką liczbą np. computerScore(2.3, 10) to przejdzie i się wysypie ;p

Reply:
Nie lepiej switcha zrobić?
```js
if (numberOfQuestions > 0) {
    switch (level) {
        case 1:
            ...
            break;
        case 2:
            ...
            break;
        default:
            throw new Error('Cośtam');
        }
}
```